lib/promutils/duration_test.go

```diff
@@ -3,13 +3,15 @@ package promutils
 import (
 	"testing"
 	"time"
+
+	"github.com/VictoriaMetrics/VictoriaMetrics/lib/timeutil"
 )
 
 func TestDuration(t *testing.T) {
```

Comment:
This test must be moved to `lib/timeutil` package

Reply:
Hey @valyala, I didn't move this test to `lib/timeutil` because it is actually testing `promutil.Duration` structure and its methods: `MarshalYAML` and `UnmarshalYAML`. While `timeutil.ParseDuration` is used as a helper.
 Do you still think it should be moved?

Reply:
In this case this test should remain here. What about adding a test for ParseDuration at lib/timeutil? This can be done in a separate commit / pull request

Reply:
See https://github.com/VictoriaMetrics/VictoriaMetrics/commit/6db97d6f79ed3427568f83ffda2e33e3a99246c2